.github/workflows/lint-do-spaces-workflow.yml

```diff
@@ -1,6 +1,9 @@
-name: CI
+name: Lint/Deploy to DigitalOcean Spaces
 
-on: push
+on:
+  push:
+    branches-ignore:
+      - master
```

Comment:
No reason to ignore master here, it may be useful to have a version that's not on gh-pages for internal testing etc.